nautobot/docs/user-guide/administration/guides/opentelemetry.md

```diff
@@ -0,0 +1,28 @@
+# OpenTelemetry
+
++++ 2.2.0
+
+Nautobot supports optionally exposing logs, traces, and metrics via [OpenTelemetry](https://opentelemetry.io/).
+
+## Configuration
+
+OpenTelemetry logs, traces, and metrics are not exposed by default. To enable this, the `OTEL_PYTHON_DJANGO_INSTRUMENT` setting should be set to `True`. This will do two things:
+
+- Logs will be exposed in the OpenTelemetry format and include the Trace ID, Span ID, and Resource attributes.
+- Metrics and Traces will be logged to console.
```

Comment:
Can you provide sample output of how things differ when this feature is enabled?